core/services/keystore/starknet.go

```diff
@@ -1,13 +1,19 @@
 package keystore
 
 import (
+	"context"
 	"fmt"
+	"math/big"
 
 	"github.com/pkg/errors"
+	"github.com/smartcontractkit/caigo"
 
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	adapters "github.com/smartcontractkit/chainlink-relay/pkg/loop/adapters/starknet"
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/starkkey"
 )
 
+//go:generate mockery --name StarkNet --output ./mocks/ --case=underscore --filename starknet.go
```

Comment:
oh is this used now? I think I just removed it or something equivalent

Reply:
yeah, i needed to pass this into the keystore refactor and so i need the mock for testing